persistence/ravendb/index_glance_raven_[6.5,).partial.md

```diff
@@ -0,0 +1,9 @@
+For a description of each feature, see the [persistence at a glance legend](/persistence/#persistence-at-a-glance).
+
+|Feature                    |   |
+|:---                       |---
+|Supported storage types    |Sagas, Outbox, Subscriptions, Timeouts
+|Transactions               |via `IDocumentSession.SaveChanges()`
+|Concurrency control        |Pessimistic concurrency, optional optimistic concurrency
```

Comment:
I thought in 6.5 optimistic concurrency was still the default but we just added a new API and marked the old one obsolete.

Reply:
What @kbaley said. We probably need the partial file to be `index_glance_raven_[6.5,7.0).partial.md` and then another partial `index_glance_raven_[7.0,).partial.md` that has "Pessimistic concurrency, optional optimistic concurrency"